packages/react-components/react-calendar-compat/library/README.md

```diff
@@ -28,7 +28,7 @@ yarn workspace @fluentui/react-calendar-compat start
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33446&commitId=99b6854fe7c9d4a6237b12a2c2f946a80b9442b6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1154| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>